vendors/espressif/boards/esp32/aws_demos/config_files/iot_pkcs11_config.h

```diff
@@ -35,6 +35,16 @@
 /* A non-standard version of C_INITIALIZE should be used by this port. */
 /* #define pkcs11configC_INITIALIZE_ALT */
 
+/**
+ * @brief ESP32 NVS Partition where PKCS #11 data is stored
+ */
+#define configPKCS11_STORAGE_PARTITION  "storage"
```

Comment:
Can you please name these using the pkcs11config* prefix, as above?

Reply:
will do. I will just point out that the file already has mixed naming conventions (right after the code I modified, there is a `configPKCS11_DEFAULT_USER_PIN` define...